src/Common/tests/Tests/System/StringTests.cs

```diff
@@ -1270,8 +1270,8 @@ public static void ContainsNoMatch_StringComparison()
                         firstSpan.ToString().StartsWith(secondSpan.ToString(), StringComparison.CurrentCulture),
                         firstSpan.Contains(secondSpan, StringComparison.CurrentCulture));
                     Assert.Equal(
-                        firstSpan.ToString().StartsWith(secondSpan.ToString(), StringComparison.CurrentCulture),
-                        firstSpan.Contains(secondSpan, StringComparison.CurrentCulture));
+                        firstSpan.ToString().StartsWith(secondSpan.ToString(), StringComparison.CurrentCultureIgnoreCase),
+                        firstSpan.Contains(secondSpan, StringComparison.CurrentCultureIgnoreCase));
```

Comment:
As long as you're fixing this, can you instead make it something like:
```C#
Assert.All(
    new[] { StringComparison.CurrentCulture, StringComparison.CurrentCultureIgnoreCase, StringComparison.InvariantCulture, StringComparison.InvariantCultureIgnoreCase },
    comparison =>
    {
        Assert.Equal(
            firstSpan.ToString().StartsWith(secondSpan.ToString(), comparison),
            firstSpan.Contains(secondSpan, comparison));
    });
```
?
That reduces the amount of code here, makes it clearer what's being tested, and helps to avoid such copy/paste issues.